pkg/controller/direct/accesscontextmanager/mappers.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +tool:fuzz-gen
+// proto.message: google.cloud.orgpolicy.v2.Policy
+// api.group: orgpolicy.cnrm.cloud.google.com
+
+package accesscontextmanager
+
+import (
+	//pb "cloud.google.com/go/accesscontextmanager/apiv1/accesscontextmanagerpb"
+	"google.golang.org/genproto/googleapis/type/expr"
+
+	krm "github.com/GoogleCloudPlatform/k8s-config-connector/apis/accesscontextmanager/v1beta1"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/controller/direct"
+)
+
+func AccessLevelExpr_FromProto(mapCtx *direct.MapContext, in *expr.Expr) krm.AccessLevelExpr {
```

Comment:
I'm surprised this one can't be auto-generated, but I'll take a look once it merges

Reply:
:shrug: